content/reference/admin/private_locations/aws/installation/index.md

```diff
@@ -31,177 +31,187 @@ Click Create bucket.
+Click on Access management > Policies, and then on Create policy.
 
-{{< img src="ecs-iam-role-1.png" alt="Choosing to create an IAM role for an Elastic Container Service Task use case" >}}
-
-Select the policies:
-
-- `AmazonS3ReadOnlyAccess` (or you can create a custom policy based on this one, to only allow access to the specific S3 bucket where you stored the Control Plane configuration)
-- You can use `AmazonEC2FullAccess` for testing, but you will most likely want to create a more restrictive policy with only the privileges needed to spawn load generator instances on EC2:
-
-```json
+`GatlingControlPlaneConfSidecarPolicy` allows sidecar to download [Control Plane configuration]({{< ref "../configuration" >}}) from S3.
+```
 {
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Action": [
-        "ec2:Describe*",
-        "ec2:CreateTags",
-        "ec2:RunInstances",
-        "ec2:TerminateInstances",
-        "ec2:AssociateAddress", <1>
-        "ec2:DisassociateAddress", <1>
-        "iam:PassRole" <2>
-      ],
-      "Effect": "Allow",
-      "Resource": "*"
-    }
-  ]
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject"
+            ],
+            "Resource": "arn:aws:s3:::{BucketName}/{ObjectName}"
+        }
+    ]
 }
 ```
 
-<1> Only required when setting elastic IP addresses on Load Generators
-<2> Only required when setting an instance profile on Load Generators
-
-If you have configured an IAM instance profile in the Control plane configuration file (with the optional key `iam-instance-profile`), you also need to add the `Action`s: `"iam:GetInstanceProfile"`, `"iam:ListInstanceProfiles"`, and `"iam:PassRole"`.
-
-Then click Next.
-
-Enter a name for the role. Add a description and tags if you want. Click on Create role.
-
-{{< img src="ecs-iam-role-2.png" alt="Reviewing the IAM role to create" >}}
-
-## ECS cluster
-
 {{< alert info >}}
-This section shows how to create a new ECS cluster, skip if you already have a cluster you want to use.
+Replace `{BucketName}` with the bucket where you uploaded the [Control Plane configuration]({{< ref "../configuration" >}}) and `{ObjectName}` with the name of the entry in the bucket.
 {{< /alert >}}
 
-In the AWS management console, from the Services menu, open Elastic Container Service (or search for "ECS" in the search bar). Click on Create Cluster. To run your containers on Fargate, choose a Networking Only cluster.
-
-{{< img src="ecs-cluster-template.png" alt="Choosing a Networking Only cluster template" >}}
-
-Then enter a cluster name (and any custom tags you may want) and click on Create.
+`GatlingControlPlaneEc2Policy` allows the Control Plane to deploy a load generator on EC2.
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "ec2:Describe*",
```

Comment:
Do we need to be more specific?

Reply:
We could try, I already fine-tune `GatlingControlPlaneConfSidecarPolicy` and `GatlingControlPlaneIAMPolicy`.
But I think we can merge this first.

Reply:
OK